junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertTrue.java

```diff
@@ -23,6 +23,12 @@
  */
 class AssertTrue {
 
+	///CLOVER:OFF
+	private AssertTrue() {
+		/* no-op */
+	}
+	///CLOVER:ON
+
```

Comment:
@sbrannen Ah, I remember now why I added these private constructors in this PR rather than a separate one in the first place. It's because error-prone purposely fails to compile whenever classes like `Assert*` don't have associated private constructors.

The error message that error-prone reports for me is as follows:
```
C:\Users\Jonathan\dev\Java\IntelliJ Projects\junit5\junit-jupiter-api\src\main\java\org\junit\jupiter\api\AssertFalse.java:24: warning: [PrivateConstructorForUtilityClass] Utility classes (only static members) are not designed to be instantiated and should be made noninstantiable with a default constructor.
class AssertFalse {
^
    (see http://errorprone.info/bugpattern/PrivateConstructorForUtilityClass)
  Did you mean to remove this line?
```

Consequently, Travis CI won't go green until https://github.com/junit-team/junit5/pull/1027 gets merged into master and I can rebase this PR on top of it.